src/cluster/cluster.h

```diff
@@ -111,4 +118,6 @@ class Cluster {
 
   std::map<int, std::string> migrated_slots_;
   std::set<int> imported_slots_;
+
+  ClusterMode cluster_mode_{ClusterMode::READONLY};
```

Comment:
These two commands should be in the **connection** dimension.

Reply:
thanks, fixed it.